packages/wmr/src/lib/rollup-plugin-container.js

```diff
@@ -290,6 +290,18 @@ export function createPluginContainer(plugins, opts = {}) {
 
 				logTransform(`${kl.dim(formatPath(id))} [${plugin.name}]`);
 				if (typeof result === 'object') {
+					if (typeof result.map === 'string') {
+						try {
+							result.map = JSON.parse(result.map);
+						} catch {
+							if (hasDebugFlag()) {
```

Comment:
No need to wrap this with `hasDebugFlag()`. That check is already part of `logTransform`.